source/backend/api/Controllers/LookupController.cs

```diff
@@ -145,6 +145,7 @@ public IActionResult GetAll()
                 var acqChecklistItemStatusTypes = _mapper.Map<Model.LookupModel[]>(_lookupRepository.GetAllAcquisitionChecklistItemStatusTypes());
                 var agreementTypes = _mapper.Map<Model.LookupModel[]>(_lookupRepository.GetAllAgreementTypes());
                 var interestHolderInterestTypes = _mapper.Map<Model.LookupModel[]>(_lookupRepository.GetAllInterestHolderInterestTypes());
+                var paymentItemTypes = _mapper.Map<Model.LookupModel[]>(_lookupRepository.GetAllPaymentItemTypes());
```

Comment:
hmm, I'd be tempted to put the word "expropriation" in here. PaymentItemTypes sounds super generic and could be in reference to a number of our entities.

Reply:
updated